spec/models/comment_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'comment should have valid attributes' do
+    post = FactoryBot.create(:post)
+    user = FactoryBot.create(:user)
+    comment = FactoryBot.build(:comment, author_id: user.id, post_id: post.id)
```

Comment:
- [ ] All your tests are running successfully as shown in the attached image below. Kindly take a look and try to fix it 👍 
![image](https://user-images.githubusercontent.com/46839548/198308902-ae1dc372-1539-4feb-b31e-403fe1e09f4f.png)